lib/model/QubitInformationObject.php

```diff
@@ -225,6 +225,50 @@ public function save($connection = null)
 
     parent::save($connection);
 
+    // Maintain siblings order when the nested set update is disabled.
+    // Add a placeholder lft value incrementing the max lft value from
+    // the resource's siblings. This placeholder will overlap with the
+    // parent's rgt value until the nested set is rebuilt.
+    if ($this->disableNestedSetUpdating && !isset($this->lft))
+    {
+      if (!isset($connection))
+      {
+        $connection = Propel::getConnection();
+      }
+
+      $connection->beginTransaction();
+
+      try
+      {
+        $sql  = 'SELECT MAX('.QubitInformationObject::LFT.')';
+        $sql .= ' FROM '.QubitInformationObject::TABLE_NAME;
+
+        if (isset($this->parentId))
+        {
+          $sql .= ' WHERE '.QubitInformationObject::PARENT_ID;
+          $sql .= ' = '.$this->parentId;
+        }
+
+        $statement = $connection->prepare($sql);
+        $statement->execute();
+        $maxLft = $statement->fetchColumn();
+  
+        $sql  = 'UPDATE '.QubitInformationObject::TABLE_NAME;
+        $sql .= ' SET '.QubitInformationObject::LFT.' = '.($maxLft + 1);
+        $sql .= ' WHERE '.QubitInformationObject::ID.' = '.$this->id;
+          
+        $connection->exec($sql);
+
+        $connection->commit();
+      }
+      catch (Exception $e)
+      {
+        $connection->rollBack();
+
+        throw $e;
```

Comment:
What state will io be in if this rollback and exception are thrown? parent::save() is called before, but all child records etc are called after. Will the io be saved, but all child recs not be created?

Reply:
@sbreker, that's a great point, thanks!

I'll move down in the method, just before indexing.